src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -511,7 +513,7 @@ namespace OpenLoco::Vehicles
         currency32_t curMonthRevenue; // 0x5E monthly revenue
         currency32_t profit[4];       // 0x62 last 4 months net profit
         uint8_t reliability;          // 0x72
-        uint8_t var_73;               // 0x73 (bit 0 = broken down, bit 1 = still powered)
+        Flags73 var_73;               // 0x73 (bit 0 = broken down, bit 1 = still powered)
```

Comment:
i think it probably makes sense to add a `has73Flags` function like `has38Flags`

Reply:
Hmm, in order to do that I should convert the Vehicle2 struct to a class or should I maintain it as a struct and implement the funciton outside? It's more a code consistency question.

Reply:
You can have functions in structs. structs/class are just identical things with different default access. You won't find many classes in this codebase as we tend to want public access to most things. In general we are thinking of removing most logic member functions from classes/structs and make it so that the only member functions are things like hasFlagsXYZ but we haven't changed to doing that yet. Long way off saying just add it to vehicle2 as a member function.

Reply:
Cool, will do, then